EIPS/eip-1422.md

```diff
@@ -0,0 +1,314 @@
+---
+eip: 1422
```

Comment:
`1422` is taken up by #1422. You could use `3664`.

Reply:
OK Thanks, will change that, so can we keep the ERC-1422 name?

Reply:
No?

Reply:
Got it, already change that, please have a look. Thanks!